packages/server/lib/util/settings.js

```diff
@@ -93,12 +123,31 @@ module.exports = {
     }, _.cloneDeep(obj))
   },
 
-  isComponentTesting (options = {}) {
-    return options.testingType === 'component'
-  },
+  configFile (projectRoot, options = {}) {
+    if (options.configFile === false) {
+      return false
+    }
+
+    if (options.configFile) {
+      return options.configFile
+    }
```

Comment:
This is the same thing.

```suggestion
    return options.configFile
```